src/sentry/options/defaults.py

```diff
@@ -43,3 +43,5 @@
 # symbolizer specifics
 register('dsym.llvm-symbolizer-path', type=String)
 register('dsym.cache-path', type=String, default='/tmp/sentry-dsym-cache')
+
+register('mail.list-namespace', type=String, default='localhost', flags=FLAG_NOSTORE)
```

Comment:
can we:

1. add doc changes in this PR
2. expose this in the mail settings admin page